frontend/src/pages/Vulnerability/Vulnerability.tsx

```diff
@@ -268,6 +269,7 @@ export const Vulnerability: React.FC = () => {
                                   <TableCell>{dom['dns-mx']}</TableCell>
```

Comment:
I think I forgot to ask about this in an earlier PR when this was introduced -- but it would be great if we could define an interface for the domain to define all its properties. This way, we don't have to cast it as `dom: any`.